README.md

```diff
@@ -1 +1 @@
-# lucasproject
\ No newline at end of file
+In this project I want to create an application that shows you and visualizes the iarquality of any city.
```

Comment:
This is a very nice idea, I think there's lots of fun things you can do with it!

For further on, you can think about adding some things to the readme file: 
- what are some functions that you will implement? 
- can you show an example of a visualization? 
- what can you do with the visualizations that are created?